src/overlays/gamestates/ovl_select/z_select.c

```diff
@@ -11,7 +11,11 @@
 
 void MapSelect_LoadTitle(MapSelectState* this) {
     this->state.running = false;
-    SET_NEXT_GAMESTATE(&this->state, ConsoleLogo_Init, ConsoleLogoState);
+    {
+        GameState* state = &this->state;
+
+        SET_NEXT_GAMESTATE(state, ConsoleLogo_Init, ConsoleLogoState);
+    }
```

Comment:
Perhaps add some TODOs here and below to update this when SET_NEXT_GAMESTATE is fixed?

Reply:
When this will be added to the macro this file won't match anymore so I thought it wasn't needed but I can add one if everyone's fine with it

Reply:
Yeah that's true. I don't mind leaving it as-is